lookout/style/format/tests/test_postprocess.py

```diff
@@ -23,6 +23,7 @@ def setUpClass(cls):
         cls.fe = FeatureExtractor(
             language=cls.language,
             **FormatAnalyzer._load_train_config({})[cls.language]["feature_extractor"])
+        cls.fe.cutoff_label_support = 0
```

Comment:
Missed this one sorry: it's better to change the config than modify the fe attribute

Reply:
it is in tests. let's keep it as is. 🙏

Reply:
fixing tests because they're not testing the behavior but use internal implementation details is tech debt and we already have lots of debt here